backend/Recodo.BlobAPI/Controllers/BlobExplorerController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Recodo.API.BLL.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Recodo.BlobAPI.Controllers
+{
+    [Route("api/Blob")]
+    [ApiController]
+    public class BlobExplorerController : ControllerBase
+    {
+        private readonly IAzureBlobService _blobService;
+
+        public BlobExplorerController(IAzureBlobService blobService)
+        {
+            _blobService = blobService;
+        }
+
+        [HttpGet("list")]
+        public async Task<IActionResult> GetBlobs()
+        {
+            return Ok(await _blobService.ListAsync());
+        }
+
+        [HttpGet]
+        public async Task<FileStreamResult> GetFile(int id)
+        {      
+            return File(await _blobService.DownloadAsync(id), "application/mp4", id.ToString()+".mp4");
```

Comment:
This should happen in the BLL layer. Just pass the data in the controller
And please use string extrapolation